sdk/keyvault/keyvault-keys/review/keyvault-keys.api.md

```diff
@@ -4,6 +4,7 @@
 
 ```ts
 
+import { AbortSignalLike } from '@azure/abort-controller';
```

Comment:
I merged a test with abortSignal and I didn't spot the missing definition in the API. I'll address this in all of the other methods in another Pull Request. I made this issue to follow up on this: https://github.com/Azure/azure-sdk-for-js/issues/4357